miscelanea/004-route-traffic/poc-004-1.yaml

```diff
@@ -0,0 +1,130 @@
+# https://kubernetes.io/docs/concepts/services-networking/service/
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: poc-004-1
+  labels:
+    app: poc-004-1
+spec:
+  serviceAccountName: poc-004
+  containers:
+    # https://github.com/nicolaka/netshoot
+    - name: client
+      image: docker.io/nicolaka/netshoot
+      command: ["cat"]
+      stdin: true
+
+    - name: http
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          add:
+            - "NET_BIND_SERVICE"
+      env:
+        - name: "ECHO_PORT"
+          value: "80"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+    - name: https
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - "NET_BIND_SERVICE"
+      env:
+        - name: "ECHO_PORT"
+          value: "443"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 443
+          hostPort: 443
```

Comment:
containerPort is the port exposed to the IP associated to the Pod.
hostPort is the port where the pod expose the port into the host/node.

Reply:
According to https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core, specifying `ports` in a `container` spec is primarily information.  Not specifying a port does not prevent it from being connected to from outside the container.

Reply:
See also the description of the `ContainerPort` object:
https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#containerport-v1-core.

Reply:
I have added the above as comments to the yaml file. Thanks.